internal/vsp/vsp.go

```diff
@@ -162,6 +162,22 @@ func (c *Client) ProcessUnprocessedTickets(ctx context.Context, policy Policy) {
 	wg.Wait()
 }
 
+// ProcessTicket ...
+func (c *Client) ProcessTicket(ctx context.Context, hash *chainhash.Hash) error {
+	// Skip tickets which have a fee tx already associated with
+	// them; they are already processed by some vsp.
+	_, err := c.Wallet.VSPFeeHashForTicket(ctx, hash)
+	if err == nil {
+		return nil
+	}
```

Comment:
are you sure about this? just because the fee was recorded as paid doesn't mean we shouldn't actively process this ticket, for example, processing is necessary to update voting preferences later on.